zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -173,8 +173,8 @@ <h3 class="interpreter-title">{{setting.name}}
       <br />
       <div class="col-md-12">
         <div class="checkbox">
-          <span class="input-group">
-            <label><input type="checkbox" style="width:0%;height:0%" id="isExistingProcess" ng-model="setting.option.isExistingProcess" ng-disabled="!valueform.$visible"/>
+          <span class="input-group" style="line-height:30px;">
+            <label><input type="checkbox" style="width:10%;height:70%" id="isExistingProcess" ng-model="setting.option.isExistingProcess" ng-disabled="!valueform.$visible"/>
```

Comment:
Okay i see. I think ```height``` can be removed but ```width``` can not be removed. 
because if remove width and height both, the screen is following like.
![image](https://cloud.githubusercontent.com/assets/3348133/17236695/b542f518-5585-11e6-9438-b9f64299f379.png)
so I think ```width``` is needed. What do you think?